app/models/katello/product.rb

```diff
@@ -258,5 +258,9 @@ def self.unused_product_id
         id
       end
     end
+
+    def self.all_products(organization_id)
+      ::Katello::Product.joins(:subscriptions => :pools).where(:organization_id => organization_id).enabled.uniq
+    end
```

Comment:
The name of this method is quite confusing I think.  

Why not just call organization.products.enabled ?  Do you need to join to subscriptions and pools?